relayer/processor/path_processor.go

```diff
@@ -598,10 +638,10 @@ func (pp *PathProcessor) processLatestMessages() error {
 	// now send messages in parallel
 	var eg errgroup.Group
 	eg.Go(func() error {
-		return pp.sendMessages(pp.pathEnd1, pathEnd1PacketMessages, pathEnd1ConnectionMessages, pathEnd1ChannelMessages)
+		return pp.sendMessages(pp.pathEnd1, pathEnd1Messages)
 	})
 	eg.Go(func() error {
-		return pp.sendMessages(pp.pathEnd2, pathEnd2PacketMessages, pathEnd2ConnectionMessages, pathEnd2ChannelMessages)
+		return pp.sendMessages(pp.pathEnd2, pathEnd2Messages)
 	})
 	return eg.Wait()
```

Comment:
If we use an error group here and messages fail to be sent on one side of the path e.g. `pathEnd1`, it could block the sending of messages on the other side of the path e.g. `pathEnd2` since afaiu the call to `eg.Wait()` will return immediately if either goroutine encounters an error.

Perhaps this is actually desirable behavior but I would imagine even if we hit an error sending messages on one side of the path it may be beneficial to still process the sending of the message on the other path end.

Reply:
Yup, errgroup will run both to completion unless the `eg.Go(...)` function argument listens to an `egCtx` or similar. 

However, errgroup will only return the first error potentially swallowing any others. In this case, I don't think it matters much, but wanted to highlight that behavior just in case.

Reply:
Sending messages is ~2 PRs out. Will keep this in mind to make sure we aren't missing any error logs.